package.json

```diff
@@ -43,13 +43,14 @@
     "test-dist": "node test/start.js test-dist",
     "test-render": "node test/start.js render",
     "test-render-react": "node test/start.js render",
-    "test-render-browser": "webpack-dev-server --env.render --progress --hot --open",
+    "test-render-browser": "webpack-dev-server --config test/webpack.config.js --env.render --progress --hot --open",
     "test-browser": "node test/start.js test-browser",
     "test-browser-local": "webpack-dev-server --config test/webpack.config.js --env.test_browser --progress --hot --open",
     "test-examples": "node test/node-examples.js",
     "test-size-es6": "npm run build-es6 && NODE_ENV=production webpack --config test/webpack.config.js --env.import-nothing --env.es6",
     "bench": "node test/start.js bench",
-    "bench-browser": "webpack-dev-server --env.bench --progress --hot --open",
+    "bench-browser": "node test/start.js bench-browser",
```

Comment:
Do we need to run bench with test?

Reply:
don't need for now, could be in future, the tests launched from test/start.js eventually will replace those launched by webpack-dev-server.